lib/xlsx/xform/sheet/sheet-view-xform.js

```diff
@@ -43,6 +43,7 @@ utils.inherits(SheetViewXform, BaseXform, {
         xmlStream.addAttribute(name, value);
       }
     };
+    add('rightToLeft', '1', model.rtl === true);
```

Comment:
Also - could you add this property to the unit test:
in spec/unit/xlsx/xform/sheet/sheet-view-xform.spec.js